### PR TITLE
AVTP pipeline documentation improvements

### PR DESCRIPTION
**Pull request by MarcinMiklasSTC**
Updated readme file for AVTP pipeline, updated API documentation and integrated into build process. Ini files improvements; added pipelines for gstreamer 1.0.

### CODE REVIEW

File: lib/avtp_pipeline/README.md [hunk 1]
```diff
@@ -4,19 +4,20 @@
...
     - MSRP
-    - igb direct for packet TX. Current linux raw sockets are used for both TX and RX.
+    - igb direct for packet TX.
     - igb credit based shaper
-    - Build system
-- Very limited testing as been performed. Primarily just the Echo talker which is a simple test stream.
+    - build system
+- Not yet integreated with OpenAVB:
+    - MAAP
+- Creadit-based shaper algorithm is not used for individual streams.
```
Comment (andrew-elder) [inline lib/avtp_pipeline/README.md]: spelling Creadit

File: run_echo_listener.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Simple script to run echo_talker
```
Comment (andrew-elder) [inline run_echo_listener.sh]: typo? echo_listener?

**Comment by andrew-elder** [comment]
This looks good to me. Anyone else?

**Comment by davidcemin** [comment]
It looks good to me.

**Pull request merged**